docs/releases.md

```diff
@@ -54,91 +75,63 @@ The first step is for the `Lead Maintainer` for `go-ipfs` to open an issue with
-
-### Release Stage 3 - Announce to the broader community
-
-- [ ] Documentation
-  - [ ] Ensure that CHANGELOG.md is up to date
-  - [ ] Ensure that README.md is up to date
-  - [ ] Ensure that all the examples we have produced for go-ipfs run without problems
-  - [ ] Update HTTP-API Documentation on the Website using https://github.com/ipfs/http-api-docs
-- [ ] Invite the community through (link to the release issue):
-  - [ ] [discuss.ipfs.io](https://discuss.ipfs.io/c/announcements)
-  - [ ] Twitter
-  - [ ] IRC
-
-### Release Stage 4 - Complete the Release
-
-- [ ] Final preparation
-  - [ ] Verify that version string in `repo/version.go` has been updated
-  - [ ] tag commit with vX.Y.Z
-  - [ ] update release branch to point to release commit (`git merge vX.Y.Z`).
-  - [ ] publish dist.ipfs.io
-  - [ ] publish next version to https://github.com/ipfs/npm-go-ipfs
-- [ ] Publish a Release Blog post (at minimum, a c&p of this release issue with all the highlights, API changes, link to changelog and thank yous)
-- [ ] Broadcasting (link to blog post)
-  - [ ] Twitter
-  - [ ] IRC
-  - [ ] Reddit
-  - [ ] [discuss.ipfs.io](https://discuss.ipfs.io/c/announcements)
-  - [ ] Announce it on the [IPFS Users mlist](https://groups.google.com/forum/#!forum/ipfs-users)
-
-# ❤️ Huge thank you to everyone that made this release possible
-
-In alphabetical order, here are all the humans that contributed to the release:
-
-- <use script -- listed in Release Stage 4 -- to generate a list of everyone that contributed for this release>
-
-# 🙌🏽 Want to contribute?
-
-Would you like to contribute to the IPFS project and don't know how? Well, there are a few places you can get started:
-
-- Check the issues with the `help wanted` label in the [go-ipfs repo](https://github.com/ipfs/go-ipfs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
-- Join an IPFS All Hands, introduce yourself and let us know where you would like to contribute - https://github.com/ipfs/team-mgmt/#weekly-ipfs-all-hands
-- Hack with IPFS and show us what you made! The All Hands call is also the perfect venue for demos, join in and show us what you built
-- Join the discussion at http://discuss.ipfs.io/ and help users finding their answers.
-- Join the [Go Core Dev Team Weekly Sync](https://github.com/ipfs/team-mgmt/issues/650) and be part of the Sprint action!
-
-# ⁉️ Do you have questions?
-
-The best place to ask your questions about IPFS, how it works and what you can do with it is at [discuss.ipfs.io](http://discuss.ipfs.io). We are also available at the `#ipfs` channel on Freenode, which is also [accessible through our Matrix bridge](https://riot.im/app/#/room/#freenode_#ipfs:matrix.org).
+
+- version string in `version.go` has been updated
+- tag commit with vX.Y.Z-rcN
+
+- [ ] Stage 1 - Internal Testing
```

Comment:
Tried to make it easier to track which stages have been completed (checking all the boxes _inside_ the stage doesn't mean that the stage itself is done).